tutorial/README.md

```diff
@@ -80,3 +80,32 @@ db.customers.insert([
 # Shut down the cluster
 docker-compose -f docker-compose-mongodb.yaml down
 ```
+
+## Using MySQL and the Avro message format
+
+To use [Avro-style messages](http://debezium.io/docs/configuration/avro/) instead of JSON,
+follow the instructions for MySQL above,
+but use the _docker-compose-mysql-avro.yaml_ configuration file instead.
+This Compose file configures the Connect service to use the Avro (de-)serializers and starts one more service,
+the Confluent service registry.
+Using Avro on conjunction with the service registry allows for much more compact messages.
+
+You can access the first version of the schema for `customers` values like so:
+
+```shell
+curl -X GET http://localhost:8081/subjects/dbserver1.inventory.customers-value/versions/1
+```
+
```

Comment:
It might be worthy to add this command too
```
curl -X GET http://localhost:8081/subjects/dbserver1.inventory.customers-value/versions/1 | jq '.schema | fromjson'
```
This prints the schema in pretty-print.

Reply:
Good idea, pushed an update. I normally use httpie instead of curl, but jq can handle the excaped JSON nicely here.